.ci/variables.sh

```diff
@@ -1,2 +1,6 @@
+#!/bin/bash
+
 export PACKAGE='github.com/m3db/m3db'
+export VENDOR_PATH=$PACKAGE/vendor
+export LICENSE_BIN=$GOPATH/src/$VENDOR_PATH/github.com/uber/uber-licence/bin/licence
```

Comment:
Kk, in future we might want to remove dependency but yeah for now I guess we just make it a requirement.